config/initializers/trestle.rb

```diff
@@ -24,7 +24,7 @@
   #
   # Set the path at which to mount the Trestle admin. Defaults to /admin.
   #
-  # config.path = "/admin"
+   config.path = "/tupress/admin"
```

Comment:
This looks like a great opportunity to use `ROOT_PATH`, though I'm not 100% sure it will be available in the initializer. That way if we deploy to `press.temple.edu` in the future, it won't break.